src/ui/layout/AboutDialog.vue

```diff
@@ -57,7 +60,7 @@
           </p>
         </div>
         <h2>Version Information</h2>
-        <ul class="t-info l-info s-info">
+        <ul data-testid="versionInfo" class="t-info l-info s-info">
```

Comment:
@ozyx not really sure what else to add here. I guess I could use an 'id'

Reply:
It's a `ul` so it implicitly has the role of `list`. I think we would just need a label.